e2etests/setup.go

```diff
@@ -239,6 +239,17 @@ func (s *hcloudK8sSetup) PrepareK8s() (string, error) {
 		return "", fmt.Errorf("%s Deploy csi: %s", op, err)
 	}
 
+	patch := `{"spec":{"template":{"spec":{"containers":[{"name":"hcloud-csi-driver","env":[{"name":"LOG_LEVEL","value":"debug"}]}]}}}}`
+	fmt.Printf("[%s] %s: Patch deployment for debug logging\n", s.MainNode.Name, op)
+	err = RunCommandOnServer(s.privKey, s.MainNode, fmt.Sprintf("KUBECONFIG=/root/.kube/config kubectl patch statefulset hcloud-csi-controller --patch '%s'", patch))
```

Comment:
Namespace should be `kube-system` (`-n kube-system` is missing here)
```suggestion
	err = RunCommandOnServer(s.privKey, s.MainNode, fmt.Sprintf("KUBECONFIG=/root/.kube/config kubectl patch statefulset hcloud-csi-controller -n kube-system --patch '%s'", patch))